cinema/serializers.py

```diff
@@ -68,13 +78,77 @@
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available"
         )
 
 
 class MovieSessionDetailSerializer(MovieSessionSerializer):
     movie = MovieListSerializer(many=False, read_only=True)
     cinema_hall = CinemaHallSerializer(many=False, read_only=True)
+    taken_places = serializers.SlugRelatedField(
+        source="tickets",
+        read_only=True,
+        many=True,
+        slug_field="full_name",
+    )
```

Comment:
The `taken_places` field in `MovieSessionDetailSerializer` uses the `full_name` property from the `Ticket` model, which is misleadingly named. Consider renaming the `full_name` property in the `Ticket` model to something more descriptive, such as `seat_location`, to better reflect its purpose.